### ISSUE TITLE
Replace Google forms for contact and VDP

### ISSUE THREAD

**Issue by michelle-rago** [cisagov/getgov#532]
The get.gov [contact form](https://docs.google.com/forms/d/e/1FAIpQLSf3UutOU4DgR2vkSdLOZk7ez9oT0cntwU9gZmrI7Zw0fC6XMg/viewform) and [VDP form](https://docs.google.com/forms/d/e/1FAIpQLSeZQGRJWsOWydQh5JU-zKqdhkXYbrY1qReCaaYGoOg14fN2iA/viewform) are both Google forms. We're using these for demonstration purposes. 

Cameron might convert these to Microsoft forms prior to launch.

**Comment by h-m-f-t** [comment]
[MSFT contact form](https://forms.office.com/g/Uq30UkMYRu)
[MSFT VDP form](https://forms.office.com/g/LUZx39aGgr)

Instead of linking, we can embed these if we choose to.

**Comment by michelle-rago** [comment]
The Google forms have been replaced with the MS forms. We do want to investigate embedding the forms and decide if we should embed the contact form on its own page (per [Slack convo](https://gsa-tts.slack.com/archives/C03QM0JGSQG/p1684426812325889)).

**Comment by gabydisarli** [comment]
@michelle-rago I can't see the linked Slack convo. Mind elaborating?

Do you have an ideas or opinions on whether these forms should be embedded or linked?

**Comment by h-m-f-t** [comment]
@gabydisarli [try this link](https://cisa-corp.slack.com/archives/C03QM0JGSQG/p1677784332398029) for that Slack convo instead?

**Comment by gabydisarli** [comment]
Thanks, I still wasn't able to see the convo via that link. But I was able to search the channel and found it that way. 

Re: embedding the form — I agree with Michelle on embedding the forms. 

Michelle, what accessibility issues have you found?

**Comment by michelle-rago** [comment]
The WebAIM WAVE tool shows a few accessibility errors with the form. I only see them when I test [the form itself](https://forms.office.com/pages/responsepage.aspx?id=bOfNPG2UEkq7evydCEI1SqHke9Gh6wJEl3kQ5EjWUKlUQzRJWDlBNTBCQUxTTzBaNlhTWURSSTBLTC4u) though. 

The WAVE tool isn't accessing the embedded form on https://federalist-877ab29f-16f6-4f12-961c-96cf064cf070.sites.pages.cloud.gov/preview/cisagov/getgov-home/mr/form-test/contact/.

**Comment by gabydisarli** [comment]
I'm seeing a11y issues on ANDI too, but, I'm not sure how "fixable" these are on the form author side. It might just be inherent to MS Forms. When trying to navigate the form using VO, I wasn't able to access anything past the title of the first question. I believe these  problems would happen whether the form is embedded or linked.

@h-m-f-t thoughts?

<img width="1332" alt="Screenshot 2023-06-08 at 3 46 50 PM" src="https://github.com/cisagov/getgov/assets/107440934/3614424e-49d6-4ac9-a77f-951858df08fa">

**Comment by michelle-rago** [comment]
We talked about adding the VDP to Github at one point. @h-m-f-t Would you prefer that we keep the VDP content on Github? We could create an issue template for reporting vulnerabilities.

**Comment by h-m-f-t** [comment]
@gabydisarli I think we should launch these but track the a11y issues as POA&Ms. It's not ideal – maybe I could help flag this to Microsoft?

@michelle-rago Having the change control for the VDP be public is welcome sign of transparency, so I'd like to have the canonical version of the VDP be on GitHub. A "friendly" version of the doc can be represented on get.gov directly, tho.

**Comment by gabydisarli** [comment]
@h-m-f-t I agree. I'll take an AI to get to the bottom of the a11y issues on the forms. Maybe it is something we can change, if not, we can launch them and make sure changing them to more accessible options is tracked in a future update. 

Re your comment to Michelle. So summarizing what you're saying here, we'll use the VDP form on the website as well as having the ability to report a vulnerability through GH.

**Comment by h-m-f-t** [comment]
To be clear, we won't "report a vulnerability through GH" but the vulnerability disclosure policy doc will be on GitHub. Actually reporting a vulnerability will take place via the Microsoft Form.

**Comment by gabydisarli** [comment]
Ah yes, I forgot the link on GH is just to view the policy and links to the form. Noted!

**Comment by gabydisarli** [comment]
@PaulKuykendall received word that these forms are actually 508 compliant, huzzah!

**Comment by gabydisarli** [comment]
Going to close this as the forms have been replaced with the appropriate MS forms and they have been reviewed and determined to be 508 compliant.
